compiler/p/codegen/BinaryEvaluator.cpp

```diff
@@ -941,7 +941,7 @@ TR::Register *OMR::Power::TreeEvaluator::imulEvaluator(TR::Node *node, TR::CodeG
 
    if (secondOp == TR::iconst || secondOp == TR::iuconst)
```

Comment:
This will only pick up constants for `imul` nodes, since `bmul` and `smul` will use `bconst` and `sconst` respectively instead. Consider switching this to:

```suggestion
    if (secondOp.isLoadConst())
```